app/google-services.json

```diff
@@ -0,0 +1,69 @@
+{
+  "project_info": {
+    "project_number": "409851798978",
+    "firebase_url": "https://oppia-android-40f74.firebaseio.com",
+    "project_id": "oppia-android-40f74",
+    "storage_bucket": "oppia-android-40f74.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:409851798978:android:d544ded26ae57f7fe6e25b",
+        "android_client_info": {
+          "package_name": "org.oppia.app"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "409851798978-4unv88bb0rj9c4ki29hanl213qg6ter8.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyDGAkJdWwOt7eLwh47YB4D-mwcZTB4nPi4"
```

Comment:
These are test keys as far as I understand so it should be safe to include them in our github repo correct? @BenHenning FYI again here we would probably have to change them to the actual release keys before releasing
@Sarthak2601 this is another param we should comment above mentioning that it needs a change on release

Reply:
@vinitamurthi can't comment as json doesn't allow that.